src/views/validated-designs/guide/index.tsx

```diff
@@ -4,56 +4,140 @@
  */
 
 import Head from 'next/head'
-import InlineLink from 'components/inline-link'
+
 import DevDotContent from 'components/dev-dot-content'
-import OutlineNav from 'components/outline-nav'
+import getDocsMdxComponents from 'views/docs-view/utils/get-docs-mdx-components'
+
+import { OutlineNavWithActive } from 'components/outline-nav/components'
+import DirectionalLinkBox from 'components/directional-link-box'
+
 import SidebarSidecarLayout from 'layouts/sidebar-sidecar'
+import {
+	SidebarHorizontalRule,
+	SidebarNavHighlightItem,
+	SidebarSectionBrandedHeading,
+} from 'components/sidebar/components'
+import SidebarNavList from 'components/sidebar/components/sidebar-nav-list'
+import SidebarBackToLink from 'components/sidebar/components/sidebar-back-to-link'
 
-import { HvdPage } from '../types'
+import s from './detail-view.module.css'
+
+import type { HvdPage } from '../types'
+import type { MDXRemoteSerializeResult } from 'next-mdx-remote'
+import type { OutlineLinkItem } from 'components/outline-nav/types'
 
 export interface ValidatedDesignsGuideProps {
 	title: string
-	// eslint-disable-next-line @typescript-eslint/no-explicit-any
-	mdxSource: any // @TODO: fix type
-	// eslint-disable-next-line @typescript-eslint/no-explicit-any
-	headers: any // @TODO: fix type
+	markdown: {
+		mdxSource: MDXRemoteSerializeResult
+		title: string
+		description: string
+	}
+	headers: OutlineLinkItem[]
 	currentPageIndex: number
-	categorySlug: string
 	basePath: string
 	pages: HvdPage[]
 }
 
 export default function ValidatedDesignGuideView({
 	title,
-	mdxSource,
+	markdown,
 	headers,
 	currentPageIndex,
 	basePath,
 	pages,
 }: ValidatedDesignsGuideProps) {
+	const docsMdxComponents = getDocsMdxComponents('hcp')
+	const backBtnTitle = 'HVD Home'
+	const backBtnUrl = basePath
+
```

Comment:
What do you think about adding variables for `pages[currentPageIndex - 1]` and `pages[currentPageIndex + 1]` since those values are used a few times?
```suggestion
const lastPage = pages[currentPageIndex - 1]
```